app/request.py

```diff
@@ -128,6 +128,13 @@ async def make_request(self):
                             trace=self.request['trace'],
                             url=self.url,
                             status_code=ex.status)
+            elif ex.status == 400:
+                logger.info('bad request',
```

Comment:
This logging level should be at least WARNING level.   RHUI is expecting the calls it makes to underlying services to succeed, and it should not be "normal" to have failures .

Reply:
Change to warn.